rustls/src/lib.rs

```diff
@@ -229,9 +229,11 @@
 //! ```
 //!
 //! # Examples
-//! [`tlsserver`](https://github.com/rustls/rustls/blob/main/examples/src/bin/tlsserver-mio.rs)
-//! and [`tlsclient`](https://github.com/rustls/rustls/blob/main/examples/src/bin/tlsclient-mio.rs)
-//! are full worked examples.  These both use mio.
+//! [`tlsserver-mio`](https://github.com/rustls/rustls/blob/main/examples/src/bin/tlsserver-mio.rs)
```

Comment:
Nit: while you're in here, add an empty line after the header?

Reply:
Done.